themes/classic/templates/catalog/_partials/product-prices.tpl

```diff
@@ -35,16 +35,12 @@
 
     {block name='product_price'}
       <div
-        class="product-price h5 {if $product.has_discount}has-discount{/if}"
-        itemprop="offers"
-        itemscope
-        itemtype="https://schema.org/Offer"
-      >
-        <link itemprop="availability" href="{$product.seo_availability}"/>
-        <meta itemprop="priceCurrency" content="{$currency.iso_code}">
+        class="product-price h5 {if $product.has_discount}has-discount{/if}">
+        <link href="{$product.seo_availability}"/>
+        <meta content="{$currency.iso_code}">
```

Comment:
That shouldn't be removed ?

Reply:
It's already present in the JSON LD declaration, do you think it's necessary to keep it in the meta ? If all other itemprop etc are removed do you think this meta with only conntent=currency is really understandable by the robots ? I readd it no problem if needed because I removed that part without special knowledge on the topic ;)